resources/META-INF/plugin.xml

```diff
@@ -29,6 +29,12 @@
       <add-to-group group-id="EditMenu" anchor="last"/>
       <keyboard-shortcut keymap="$default" first-keystroke="shift ctrl alt A"/>
     </action>
+    <action id="one.util.ideaplugin.screenshoter.SaveImageAction" 
+            class="one.util.ideaplugin.screenshoter.SaveImageAction"
+            text="Save as image" description="Save the selection as image">
+      <add-to-group group-id="EditMenu" anchor="last"/>
+      <keyboard-shortcut keymap="$default" first-keystroke="shift ctrl alt S"/>
```

Comment:
By default this is a "project structure" shortcut (at least on Windows) which is pretty useful. I don't think we should conflict with it. Probably we can leave this action without shortcut by default?

Reply:
agree